src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/register/DetachedRegisterPage.java

```diff
@@ -0,0 +1,80 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth.register;
+
+import com.wikia.webdriver.pageobjectsfactory.componentobject.modalwindows.FacebookSignupModalComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.DetachedWindow;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.FacebookAuthContext;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.DetachedSignInPage;
+
+public class DetachedRegisterPage extends DetachedWindow implements RegisterPage,
+  FacebookAuthContext {
+
+  private AttachedRegisterPage registerPage;
+  private final static String TITLE = "Join Fandom Today";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)